span-stacktrace/README.md

```diff
@@ -40,6 +40,16 @@ SdkTracerProvider tracerProvider = SdkTracerProvider.builder()
 OpenTelemetrySdk sdk = OpenTelemetrySdk.builder().setTracerProvider(tracerProvider).build();
 ```
 
+### Configuration
+
+Even if autoconfiguration is not yet supported, usages of this module must use the
+`otel.span.stacktrace.min.duration` configuration option (in nanoseconds, defaults to 5ms) to
```

Comment:
nanoseconds, is that intended?

Reply:
Yes, the span duration (latency) is already measured in nanoseconds so it's simpler to keep the same unit.

Reply:
In fact here when the value is parsed as `java.time.Duration` from configuration it can be set to any unit, so only the default value unit needs to be specified.